storage/innobase/xtrabackup/test/suites/keyring/redo_log_archive.sh

```diff
@@ -0,0 +1,71 @@
+#
+# PXB-1942: Backup fails when database is on a fast storage with redo archive logs enabled
+#
+. inc/common.sh
+. inc/keyring_file.sh
+
+require_server_version_higher_than 8.0.16
```

Comment:
any reason for the 8.0.16 minimum version?

Reply:
redo archive feature was added in 8.0.16